resources/devbox/macos/macos.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+## Note: does not use standard build script include
+# adjust relative paths as necessary. Note that this will not work in a symlinked path reliably
+THIS_SCRIPT="${BASH_SOURCE[0]}"
+THIS_DIR=$(dirname "$THIS_SCRIPT")
+
+# Determine which components to install
+REQUIRE_ANDROID=false
+REQUIRE_IOS=false
+REQUIRE_MACOS=false
+REQUIRE_WEB=false
+
+# Optional components
+REQUIRE_KMCOMP=false
+REQUIRE_PANDOC=false
+REQUIRE_SENTRYCLI=false
+
+# Parse args
+shopt -s nocasematch
+
+while [[ $# -gt 0 ]] ; do
+    key="$1"
```

Comment:
If no command line arguments are given the script will still fall through to the message about configuring your system and will also attempt to update the bashrc which will fail as ".cargo/env" etc do not exist as they were not installed.